legal-api/src/legal_api/services/filings/validations/alteration.py

```diff
@@ -84,7 +84,7 @@ def company_name_validation(filing):
 
         error_msg = """The name type associated with the name request number entered cannot be used for this
                        transaction type."""
-        if not nr_response['requestTypeCd'] in ('CCR', 'CCP', 'BEC', 'BECV'):
+        if not nr_response['requestTypeCd'] in ('CCR', 'CCP', 'BEC', 'BECV', 'CCV', 'UC', 'ULCB', 'ULCE'):
```

Comment:
![image](https://github.com/bcgov/lear/assets/3484109/66a3444c-015d-4c8b-9bcc-7b2a08275319)

![image](https://github.com/bcgov/lear/assets/3484109/caa4c93f-5576-4152-a785-8da9355282cb)

insert into request_type (REQUEST_TYPE_CD,SHORT_DESC ,FULL_DESC ,CLASS_TYPE_CD)
values ('ULCB' ,'Unlimited Liability Company to BC Conversion','Unlimited Liability Company to BC Conversion','CORP' );

insert into request_type (REQUEST_TYPE_CD,SHORT_DESC ,FULL_DESC ,CLASS_TYPE_CD)
values ('ULBE' ,'Unlimited Liability Company to Benefit Company Conversion','Unlimited Liability Company to Benefit Company Conversion','CORP' );